tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -111,4 +186,119 @@ public function generateOrderInvoice($orderReference)
             new GenerateInvoiceCommand((int) $order->id)
         );
     }
+
+    /**
+     * @When I update orders with ids :references status to :status
+     *
+     * @param string $orderIdsString
+     * @param string $status
+     *
+     * @throws OrderException
+     * @throws RuntimeException
+     */
+    public function iUpdateOrdersWithIdsStatusTo(string $orderIdsString, string $status)
```

Comment:
Just a suggestion, but nothing important: it's better to have "classical" function names for behat steps
```suggestion
    public function updateOrdersToStatuses(string $orderIdsString, string $status)
```

because steps can call other steps, so you can create macro steps:

```
    public function iCreateAFullOrder()
    {
        $this->createCustomer();
        $this->createAddress();
        ...
    }
```